src/Altinn.Platform/Altinn.Platform.Storage/Storage/Controllers/InstancesController.cs

```diff
@@ -581,6 +583,41 @@ public async Task<ActionResult<Instance>> Delete(int instanceOwnerPartyId, Guid
             return Ok(updatedInstance);
         }
 
+        /// <summary>
+        /// Updates the presentation fields of an instance
+        /// </summary>
+        [Authorize(Policy = AuthzConstants.POLICY_INSTANCE_WRITE)]
+        [HttpPut("{instanceOwnerPartyId:int}/{instanceGuid:guid}/presentationFields")]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        public async Task<Instance> UpdatePresentationFields(
```

Comment:
We should add the following attributes. 
```
        [Consumes("application/json")]
        [Produces("application/json")]
```
This will effectively remove support for xml and plain text requests and responses.